currencyConverter.js

```diff
@@ -0,0 +1,57 @@
+var request = require('request');
+var cheerio = require('cheerio');
+
+/**
+* Rounds off currency output 2 decimal places
+*/
+
```

Comment:
You can remove lines 7, 19 and 32. So that there is no gap between the doc-block, and the function it is referring to.